docs/index.rst

```diff
@@ -9,15 +9,15 @@ Welcome to disnake
 .. image:: /images/snake.svg
 .. image:: /images/snake_dark.svg
 
-disnake is a modern, easy to use, feature-rich, and async ready API wrapper
+disnake is a modern, easy-to-use, feature-rich, and async-ready API wrapper
```

Comment:
Not entirely sold on the easy-to-use changes here.

Reply:
+1, "easy to use" seems better here

Reply:
@toolifelesstocode would you please update this?

Reply:
All fixed.